contracts/asset-proxy/contracts/src/ERC1155Proxy.sol

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
How this already been reviewed elsewhere? Are we duplicating Solidity source code within the monorepo?

Reply:
Yup! Been reviewed in #1661. Just had to do a cheeky rebase.